parcels/particleset.py

```diff
@@ -29,13 +29,13 @@ class ParticleSet(object):
     :param depth: Optional list of initial depth values for particles. Default is 0m
     :param time: Optional list of initial time values for particles. Default is fieldset.U.grid.time[0]
     :param repeatdt: Optional interval (in seconds) on which to repeat the release of the ParticleSet
-    :param coordinates_var_precision: Floating precision for lon, lat, depth particle coordinates.
+    :param lonlatdepth_dtype: Floating precision for lon, lat, depth particle coordinates.
            It is either 'single' or 'double'. Default is 'single' if fieldset.U.interp_method is 'linear'
```

Comment:
Shouldn't we then also rename the options to `np.float32` and `np.float64`?

Reply:
Done